language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -73,7 +100,7 @@ daml2ts Options{..} pkgId pkg mbPkgName = do
     forM_ (packageModules pkg) $ \mod -> do
         whenJust (genModule pkgId mod) $ \modTxt -> do
             let outputFile = outputDir </> joinPath (map T.unpack (unModuleName (moduleName mod))) FP.<.> "ts"
-            putStrLn $ "Generating " ++ outputFile
+            -- putStrLn $ "Generating " ++ outputFile
```

Comment:
Either remove it or leave it in but commenting it out seems weird.

Reply:
Sure. Will do.